src/app/components/header/userMenu/userMenu.component.ts

```diff
@@ -1,16 +1,18 @@
-import { Component } from '@angular/core';
-import { AuthStore } from '../../../stores/auth.store';
-import { AuthService } from '../../../services/api/auth.service';
+import { Component, Input } from '@angular/core';
+import { AuthService } from '../../../services/auth.service';
+import { Role } from '../../../constants/roles';
 
 @Component({
   selector: 'rm-header-user-menu',
   templateUrl: './userMenu.component.html',
 })
 export class UserMenuComponent {
-  constructor(
-    private authStore: AuthStore,
-    private authService: AuthService,
-  ) {}
+  @Input()
+  links = [];
+
+  Role = Role;
+
+  constructor(public authService: AuthService) {}
 
   logout() {
     this.authService.logout();
```

Comment:
(IMO) After doing logout, I think you should navigate to home page. For example, if you are in admin page and do logout, you have to exit of admin pages

Reply:
Done